### PR TITLE
fix: use modal dialog for user sign in

### PR DESCRIPTION
**Pull request by HuihuiWu-Microsoft**
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/9888588

### CODE REVIEW

File: packages/vscode-extension/src/resources/Strings.json [hunk 1]
```diff
@@ -76,7 +76,7 @@
             "prepareTask": " Prepare task."
         },
         "appStudioLogin": {
-            "warningMsg": "The Teams Toolkit requires a Microsoft 365 organizational account where Teams is running and has been registered. You can create a free testing account from [M365 Developer Program](https://developer.microsoft.com/en-us/microsoft-365/dev-program) if needed.",
+            "warningMsg": "The Teams Toolkit requires a Microsoft 365 organizational account where Teams is running and has been registered. You can create a free testing account from M365 Developer Program if needed.",
```
Comment (HuihuiWu-Microsoft) [inline packages/vscode-extension/src/resources/Strings.json:79]: Modal dialog doesn't support markdown, so remove the link here

**Review by nliu-ms — APPROVED** [review]

**Review by xiaolang124 — APPROVED** [review]

**Review by swatDong — APPROVED** [review]
The code logic looks good to me.

File: packages/vscode-extension/src/resources/Strings.json (hunk 1 above)
Comment (swatDong) [reply packages/vscode-extension/src/resources/Strings.json:79]: BTW, should here be `infoMsg`? Since it's `showInformationMessage`.

Comment (HuihuiWu-Microsoft) [reply packages/vscode-extension/src/resources/Strings.json:79]: Sure

File: packages/vscode-extension/src/resources/Strings.json [hunk 2]
```diff
@@ -74,11 +74,11 @@
       "prepareTask": " Prepare task."
     },
     "appStudioLogin": {
-      "warningMsg": "The Teams Toolkit requires a Microsoft 365 organizational account where Teams is running and has been registered. You can create a free testing account from [M365 Developer Program](https://developer.microsoft.com/en-us/microsoft-365/dev-program) if needed.",
+      "message": "The Teams Toolkit requires a Microsoft 365 organizational account where Teams is running and has been registered. You can create a free testing account from M365 Developer Program if needed.",
```
Comment (1openwindow) [inline packages/vscode-extension/src/resources/Strings.json:77]: @timngmsft  for awareness

**Review by 1openwindow — APPROVED** [review]

**Review by timngmsft — APPROVED** [review]

**Review by 1openwindow — APPROVED** [review]

**Pull request merged**